File: Pages/CallDotNet1.razor.js
export async function setMessage() {
  const { getAssemblyExports } = await globalThis.getDotnetRuntime(0);
  var exports = await getAssemblyExports("BlazorWASM70UnmarshalledJS.dll");

  document.getElementById("result").innerText = 
    exports.BlazorWASM70UnmarshalledJS.Pages.CallDotNet1.GetMessageFromDotnet();
}

File: Pages/CallDotNet1.razor.cs
using System.Runtime.InteropServices.JavaScript;
using System.Runtime.Versioning;

namespace BlazorWASM70UnmarshalledJS.Pages
{
    [SupportedOSPlatform("browser")]
    public partial class CallDotNet1
    {
        [JSImport("setMessage", "CallDotNet1")]
        internal static partial void SetWelcomeMessage();

        [JSExport]
        internal static string GetMessageFromDotnet()
        {
            return "¡Hola desde Blazor!";
        }
    }
}
